examples/acquiring_data/goes_xrs_example.py

```diff
@@ -89,6 +89,12 @@
 fig, ax = plt.subplots()
 ax.plot(goes_flare.index, np.gradient(goes_flare.quantity("xrsb")))
 ax.set_ylabel("Flux (Wm$^{-2}$$s^{-1}$)")
+# You may notice that the time axis in the above example is not really well laid out.
+# We can do the following to fix that, this is done by the ``plot`` method.
```

Comment:
What do you mean by "this is done by the plot method"? I see that this code is now included in the `plot` methods for each source. I wonder if we should be less opinionated in `plot()` and then just have this example here showing how to fix the labels in the case where they're not aesthetically pleasing?

Reply:
Not sure really.

Reply:
I removed all this.